RationaleMCP/0034/ternary.md

```diff
@@ -32,10 +32,12 @@ There is a total order of the possible `Ternary` values given by `Ternary(false)
 
 ## Expressions with Ternary
 
-Logical operations on `Ternary` are defined according to Kleene, where `unknown` can be thought of as representinc uncertainty about being `true` or `false`.  With this interpretation the Kleene logic tables then follow from the Boolean logic.
+The logical connectives `not`, `and` and `or` on `Ternary` are defined in accordance with Kleene, where `unknown` can be thought of as representinc uncertainty about being `true` or `false`.  With this interpretation the truth tables then follow from the Boolean logic.
```

Comment:
Regarding comment in main MCP issue:
>> Right. We should not use Ternary to build theorem proofers.
>
> But our users will see the 3-valued logic, and start trying it on more complex domains than just dialog visibility. I have this requirements engineering library of EDF in my mind that had been proposed some time ago, also as a motivation to add 3-valued logic (at the end it turned out it was kind of a 4-valued logic because of the need to model time ... voilà, there it goes). With the proposed 3-valued logic, this requirements engineering example will first become sort-of more easy to develop, but then turns out to fall short and then all the craziness comes in step by step as additional requirements and corner-cases.

Your comment makes me think of the connectives I would have liked to handle things like "Boolean annotations with clever automatic default".  Instead of interpreting `unknown` as unknown, I would have liked it to be interpreted as _disregard_, allowing it to be cancelled from both conjunction and disjunction.  That is, instead of `and` and `or`, I would have wanted the connectives `duand` (disregard-unknown-and) and `duor` (disregard-unknown-and).  Do you happen to know if this has a name in the literature?  If so, maybe this could also be mentioned non-normatively, as it illustrates that not even the truth tables for `and` and `or` are as obvious as one might first think?